pkg/libmount/filesystem.go

```diff
@@ -0,0 +1,134 @@
+/*
+Copyright 2020 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package libmount
+
+import "strings"
+
+type Filesystem struct {
```

Comment:
Can you refer the util-linux package here, we need to give proper attribution that this package was inspired from util-linux libmount.